BuildResidentialHPXML/measure.rb

```diff
@@ -3214,6 +3216,12 @@ def argument_errors(args)
     error = (args[:heating_system_type] == 'none') && (args[:heat_pump_type] == 'none') && (args[:heating_system_2_type] != 'none')
     errors << 'A second heating system was specified without a primary heating system.' if error
 
+    if (args[:heat_pump_backup_type] == HPXML::HeatPumpBackupTypeSeparate) && [HPXML::HVACTypeFurnace, HPXML::HVACTypeFixedHeater].include?(args[:heating_system_2_type]) # separate ducted backup
```

Comment:
A fixed heater is not ducted, it's the same as a portable heater except can't be moved around.